pkg/new-ui/v1beta1/frontend/src/app/pages/experiment-creation/hyper-parameters/add-modal/add-modal.component.html

```diff
@@ -0,0 +1,81 @@
+<h1 mat-dialog-title>Add new parameter</h1>
```

Comment:
Do you need this add modal since you have the common one: https://github.com/kubeflow/katib/tree/1b48a137ac24428d83bd411bc1ce4d3f4724d1c6/pkg/new-ui/v1beta1/frontend/src/app/shared/params-list/add-modal ?

Reply:
Ahh, indeed I should have used the shared modal here.
The shared modal was created based on that initial modal but Ι forgot to remove it from there. Pushed a commit